libs/designer-ui/src/lib/floatingactionmenu/floatingactionmenuoutputs/index.tsx

```diff
@@ -61,6 +63,15 @@ export const FloatingActionMenuOutputs = (props: FloatingActionMenuOutputsProps)
     }
   };
 
+  const onDynamicallyAddedParameterDescriptionChange = (schemaKey: string, newValue: string | undefined): void => {
+    const { onChange } = props;
+    if (onChange) {
+      const viewModel = clone(props.editorViewModel);
+      viewModel.schema.properties[schemaKey].description = newValue || '';
```

Comment:
I'd advise optionally chaining here, given that the editorViewModel can be of type any, if there is a chance that some part of viewModel.schema.properties[schemaKey].description is undefined, this would crash the designer

Reply:
> I'd advise optionally chaining here, given that the editorViewModel can be of type any, if there is a chance that some part of viewModel.schema.properties[schemaKey].description is undefined, this would crash the designer

Hey, I just went through and generally cleaned up the entire component with undefined guards when working with that editor view model, here and elsewhere, let me know if my solution works for you